### PR TITLE
editor-stepping: Allow changing highlight color

### LINKED ISSUE #2957: [editor-stepping] Ability to change the highlighted color
### Is your request related to a problem? Please describe.
I don't want blue to be the only color available for the highlighting blocks addon.

### Describe the solution you'd like

Setting to change color for the highlight in the editor-stepping addon

### PR DESCRIPTION
**Pull request by CST1229**
Resolves #2957

### Changes

Adds a color option to recolor the border in the editor-stepping (Highlight currently executing blocks) addon. Also slightly rewords the addon to not use the word blue. The default color is still #0000FF blue.

### Reason for changes

Some people may not like the blue or think it's too strong:
>**FunctionalMetatable wrote:**
>
>I don't want blue to be the only color available for the highlighting blocks addon.

### Tests

Tested on Firefox, should work on Chrome.
redgreenorangeblackwhite

### CODE REVIEW

File: addons/editor-stepping/userscript.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ export default async function ({ addon, global, console }) {
       </feFuncA>
     </feComponentTransfer>
 
-    <feFlood flood-color="blue" flood-opacity="1" result="outColor">
+    <feFlood flood-color="` + addon.settings.get("highlight-color")  + `" flood-opacity="1" result="outColor">
```
Comment (RedGuy12) [inline addons/editor-stepping/userscript.js:18]: Do we need to worry about xss here?

**Review by RedGuy12 — CHANGES_REQUESTED** [review]

File: addons/editor-stepping/userscript.js (hunk 1 above)
Comment (apple502j) [reply addons/editor-stepping/userscript.js:18]: Yes.

**Comment by apple502j** [comment]
Cannot merge due to sec regressions

File: addons/editor-stepping/userscript.js (hunk 1 above)
Comment (CST1229) [reply addons/editor-stepping/userscript.js:18]: It's just a color input, however modifying the config file could do something here. But I don't think it's such a big thing to worry about

**Comment by CST1229** [comment]
> 
> 
> Cannot merge due to sec regressions

Can you explain?

File: addons/editor-stepping/userscript.js (hunk 1 above)
Comment (apple502j) [reply addons/editor-stepping/userscript.js:18]: We won't merge this until this is fixed.

**Comment by WorldLanguages** [comment]
IMO xss isn't a big deal here...
Update: recent commit is cleaner code and "safer" so problem solved.

**Comment by CST1229** [comment]
> IMO xss isn't a big deal here...
Yeah I was thinking that, it's literally a color input.

**Comment by WorldLanguages** [comment]
Well, it's most likely nothing bad would happen with the previous version, but a chain of bugs could happen and an addon doing this could be the last one on the chain

**Comment by WorldLanguages** [comment]
You can enable Prettier in your fork's action tab, or alternatively tell me when this is ready to review and I'll lint myself

**Review by WorldLanguages — APPROVED** [review]

**Pull request merged**